packages/kit/src/runtime/client/renderer.js

```diff
@@ -104,11 +147,26 @@ export class Renderer {
 			promise: null
 		};
 
+		const versionPollInterval =
+			typeof import.meta.env.VITE_APP_VERSION_POLL_INTERVAL === 'string'
+				? parseInt(import.meta.env.VITE_APP_VERSION_POLL_INTERVAL)
+				: 60_000;
```

Comment:
as long as we set `process.env.VITE_APP_VERSION_POLL_INTERVAL` in dev as well as build (albeit to 0, i.e. no polling) we can simplify this (note that it also changes `versionPollInterval` to `version_poll_interval` to be consistent with internal variable names elsewhere in the codebase):

```suggestion
		const version_poll_interval = +import.meta.env.VITE_APP_VERSION_POLL_INTERVAL;
```

Reply:
Gives type error now, too.